helm/alfresco-content-services/templates/_helpers-persistence.tpl

```diff
@@ -0,0 +1,36 @@
+{{- define "data_volume" -}}
+{{- $svc_name := .service.name -}}
+{{- with .persistence -}}
+{{- $sc_name := .storageClass.name | default "default" -}}
+- name: data
+  persistentVolumeClaim:
+    claimName: >-
+      {{ .existingClaim | default (printf "%s-%s-pvc" $svc_name $sc_name ) }}
+{{- end -}}
+{{- end -}}
+
+{{- define "component_pvc" -}}
+{{ $svc_name := .service.name }}
+{{- with .persistence }}
+{{- $sc_name := .storageClass.name | default "default" -}}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ printf "%s-%s-pvc" $svc_name $sc_name }}
+spec:
+  {{- if .storageClass.enabled }}
+  volumeMode: {{ .storageClass.volumeMode | default "Filesystem" }}
+  accessModes:
+  {{- range .storageClass.accessModes }}
+    - {{ . }}
+  {{- end }}
+  {{- if .storageClass.name }}
+  storageClassName: {{ .storageClass.name }}
+  {{- end }}
+  {{- end }}
+  resources:
+    requests:
+      storage: {{ .baseSize | default "20Gi" }}
+{{- end }}
+{{- end -}}
```

Comment:
should be good to have in alfresco-common?

so we can use it also in aps chart

Reply:
Of course yes. But for now it'll be only used in the main chart. The one who needs it move it to alfresco-common ;)

Reply:
I guess who will be then... =D